Pinetree/wwwroot/lib/KaTeX/contrib/auto-render.js

```
@@ -0,0 +1,349 @@
+  }
+};
+
+var renderMathInElement = function renderMathInElement(elem, options) {
+  if (!elem) {
+    throw new Error("No element provided to render");
+  }
+
+  var optionsCopy = {}; // Object.assign(optionsCopy, option)
+
+  for (var option in options) {
+    if (options.hasOwnProperty(option)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)